src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Services/Interface/IProcess.cs

```diff
@@ -11,6 +16,32 @@ public interface IProcess
         /// Returns a stream that contains the process definition.
         /// </summary>        
         /// <returns>the stream</returns>
-        Stream GetProcessDefinition(string org, string app);
+        Stream GetProcessDefinition();
+
+        /// <summary>
+        /// Starts the process with a given valid startEvent. If process is allready started the current instance is returned.
+        /// </summary>
+        /// <param name="instance">instance to start process on</param>
+        /// <param name="validStartEvent">valid start event identifier</param>
+        /// <returns>instance with started process</returns>
+        Task<Instance> ProcessStart(Instance instance, string validStartEvent, UserContext userContext);
+
+        /// <summary>
+        /// Starts the process and moves it to its first Task. If process is allready started operation is ignored.
```

Comment:
comment mentions first task and function name refereres to next task.. Could we change one or the other to match what is actually done?